content/en/blog/2024/otel-errors/index.md

```diff
@@ -0,0 +1,291 @@
+is a structured log message embedded within a span. Span events help enhance spans 
+by providing descriptive information about a span. [Span events can also have 
+attributes of their own](https://opentelemetry-python.readthedocs.io/en/latest/_modules/opentelemetry/trace/span.html#NonRecordingSpan.add_event). 
+
+When a span status is set to `Error`, a span event is created automatically, 
+capturing the span’s resulting error message and stack trace as an event on that 
+span. You can further enhance this span error by adding attributes to it.
+
+Earlier, we mentioned a method called `RecordException`. Per the spec (emphasis 
+our own), “To facilitate recording an exception languages SHOULD provide a 
+RecordException method **if the language uses exceptions**… The signature of the 
+method is to be determined by each language and can be overloaded as appropriate.” 
+
+Since Go doesn’t support the “conventional” concept of exceptions, it instead 
+supports [`RecordError`](https://github.com/open-telemetry/opentelemetry-go/blob/main/sdk/trace/span.go#L445-L467), which essentially does the same thing idiomatically. You have to 
+make an additional call to set its status to `Error` if that’s what it should be, 
+as it won’t automatically be set to that. Similarly, `RecordException` can be used 
+to record span events without setting the span’s status to `Error`, which means you 
+can use it to record any additional data about a span. 
+
+By decoupling the span status from being automatically set to `Error` when a span 
+exception occurs, you can support the use case where you can have an exception 
+event with a status of `Ok` or `Unset`. This gives instrumentation authors the 
+most flexibility. 
+
+### Errors Depicted in Logs
+In OTel, a log is a structured, timestamped message emitted by a service or other 
+component. The recent addition of logs to OTel gives us yet another way of 
+reporting errors. Logs have traditionally had different severity levels for 
+representing the type of message being emitted such as `DEBUG`, `INFO`, `WARNING`, 
+`ERROR`, and `CRITICAL`.
+
+OTel allows for the correlation of logs to traces, in which a log message can be 
+associated to a span within a trace, via trace context correlation. Hence, looking 
+for a log message with a log level of `ERROR` or `CRITICAL` can yield further 
+information of what led to that error, by pulling up the correlated trace.
+
+To record an error on a log, either `exception.type` or `exception.message` is 
+required, while `exception.stacktrace` is recommended. You can view more 
+information about the semantic conventions for log exceptions [here](/docs/specs/semconv/exceptions/exceptions-logs/).
+
+## Logs or spans to capture errors?
+
+After all this, you might be wondering which signal to use to capture errors: 
+spans or logs? The answer is: "It depends!" Perhaps your team primarily uses 
+traces; perhaps they primarily use logs.
```

Comment:
Just a quick suggestion to make this sentence parallel.

```suggestion
spans or logs? The answer is: "It depends!" Perhaps your team primarily uses 
traces; perhaps it primarily uses logs.
```

Reply:
Thank you! Great catch.